.stickler.yml

```diff
@@ -0,0 +1,4 @@
+linters:
```

Comment:
.stickler.yml file should not be a part of commit and this PR.

> Zsynchronizujcie swoje branche z masterem głównego repo, bo żeby to działało, w roocie na branchu, z którego robicie PR musicie mieć pliczek *.stickler.yml*.

So please remove this file from this PR and before next lesson please sync your fork with upstream. Details are in "GIT training materials"